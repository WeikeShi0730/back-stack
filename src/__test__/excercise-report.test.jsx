import { render } from "@testing-library/react";
import ExcerciseReport from "../pages/excercise-report/excercise-report.component";

describe("<ExcerciseReport />", () => {
  test("<ExcerciseReport />", () => {
    const { baseElement } = render(<ExcerciseReport />);
    expect(baseElement).toMatchSnapshot();
  });
});
