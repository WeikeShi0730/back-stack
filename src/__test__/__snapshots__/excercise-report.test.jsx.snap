// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExcerciseReport /> <ExcerciseReport /> 1`] = `
<body>
  <div>
    <div
      class="h-full"
    >
      <div
        class="flex flex-col lg:flex-row justify-center items-center m-5"
      >
        <div
          class="mx-3 my-1 w-2/3 md:w-1/3 lg:w-80 css-b62m3t-container"
        >
          <span
            class="css-1f43avz-a11yText-A11yText"
            id="react-select-data-live-region"
          />
          <span
            aria-atomic="false"
            aria-live="polite"
            aria-relevant="additions text"
            class="css-1f43avz-a11yText-A11yText"
          />
          <div
            class=" css-1yqfws-control"
          >
            <div
              class=" css-319lph-ValueContainer"
            >
              <div
                class=" css-14el2xx-placeholder"
                id="react-select-data-placeholder"
              >
                Date...
              </div>
              <div
                class=" css-6j8wv5-Input"
                data-value=""
              >
                <input
                  aria-autocomplete="list"
                  aria-controls="react-select-data-listbox"
                  aria-describedby="react-select-data-placeholder"
                  aria-expanded="false"
                  aria-haspopup="true"
                  aria-owns="react-select-data-listbox"
                  autocapitalize="none"
                  autocomplete="off"
                  autocorrect="off"
                  class=""
                  id="react-select-data-input"
                  role="combobox"
                  spellcheck="false"
                  style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                  tabindex="0"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class=" css-1hb7zxy-IndicatorsContainer"
            >
              <span
                class=" css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                class=" css-tlfecz-indicatorContainer"
              >
                <svg
                  aria-hidden="true"
                  class="css-tj5bde-Svg"
                  focusable="false"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mx-3 my-1 w-2/3 md:w-1/3 lg:w-48 css-b62m3t-container"
        >
          <span
            class="css-1f43avz-a11yText-A11yText"
            id="react-select-session-live-region"
          />
          <span
            aria-atomic="false"
            aria-live="polite"
            aria-relevant="additions text"
            class="css-1f43avz-a11yText-A11yText"
          />
          <div
            class=" css-1yqfws-control"
          >
            <div
              class=" css-319lph-ValueContainer"
            >
              <div
                class=" css-14el2xx-placeholder"
                id="react-select-session-placeholder"
              >
                Start time...
              </div>
              <div
                class=" css-6j8wv5-Input"
                data-value=""
              >
                <input
                  aria-autocomplete="list"
                  aria-controls="react-select-session-listbox"
                  aria-describedby="react-select-session-placeholder"
                  aria-expanded="false"
                  aria-haspopup="true"
                  aria-owns="react-select-session-listbox"
                  autocapitalize="none"
                  autocomplete="off"
                  autocorrect="off"
                  class=""
                  id="react-select-session-input"
                  role="combobox"
                  spellcheck="false"
                  style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                  tabindex="0"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class=" css-1hb7zxy-IndicatorsContainer"
            >
              <span
                class=" css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                class=" css-tlfecz-indicatorContainer"
              >
                <svg
                  aria-hidden="true"
                  class="css-tj5bde-Svg"
                  focusable="false"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mx-3 my-1 w-2/3 md:w-1/3 lg:w-48 css-b62m3t-container"
        >
          <span
            class="css-1f43avz-a11yText-A11yText"
            id="react-select-driver-live-region"
          />
          <span
            aria-atomic="false"
            aria-live="polite"
            aria-relevant="additions text"
            class="css-1f43avz-a11yText-A11yText"
          />
          <div
            class=" css-1yqfws-control"
          >
            <div
              class=" css-319lph-ValueContainer"
            >
              <div
                class=" css-14el2xx-placeholder"
                id="react-select-driver-placeholder"
              >
                End time...
              </div>
              <div
                class=" css-6j8wv5-Input"
                data-value=""
              >
                <input
                  aria-autocomplete="list"
                  aria-controls="react-select-driver-listbox"
                  aria-describedby="react-select-driver-placeholder"
                  aria-expanded="false"
                  aria-haspopup="true"
                  aria-owns="react-select-driver-listbox"
                  autocapitalize="none"
                  autocomplete="off"
                  autocorrect="off"
                  class=""
                  id="react-select-driver-input"
                  role="combobox"
                  spellcheck="false"
                  style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                  tabindex="0"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class=" css-1hb7zxy-IndicatorsContainer"
            >
              <span
                class=" css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                class=" css-tlfecz-indicatorContainer"
              >
                <svg
                  aria-hidden="true"
                  class="css-tj5bde-Svg"
                  focusable="false"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="flex justify-center items-center"
        >
          <div
            class="w-11/12 h-full"
          >
            <div
              class="recharts-responsive-container my-5"
              style="width: 100%; height: 400px;"
            />
            <div
              class="recharts-responsive-container my-5"
              style="width: 100%; height: 400px;"
            />
          </div>
        </div>
      </div>
      <div
        class="flex justify-center items-center"
      >
        <div
          class="flex flex-col w-11/12"
        >
          <div
            class="text-sm md:text-base lg:text-lg text-center m-5"
          >
            Comparison
          </div>
          <div
            class="recharts-responsive-container my-5"
            style="width: 100%; height: 100px;"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
